source/PreferencesPanel.cpp

```diff
@@ -807,10 +744,24 @@ void PreferencesPanel::DrawSettings()
 			text = isOn ? "on" : "off";
 
 		if(setting == hoverItem)
+		{
+			table.SetHighlight(-120, 120);
 			table.DrawHighlight(back);
+		}
```

Comment:
The mouse hovering is behaving funky in the settings. I sent a video on Discord showing what I mean.